product/tests/test_product.py

```diff
@@ -24,12 +25,19 @@ def saved_product0(product0):
 
 class TestProductModel:
     @pytest.mark.django_db()
-    def test_save_product_and_delete_product(self, saved_product0):
+    def test_save_product(self, saved_product0):
+        temp = Product(saved_product0.qr_code, saved_product0.product_name, saved_product0.description)
         assert saved_product0 in Product.objects.all()
-        Product.delete_product(saved_product0.qr_code)
+        saved_product0.delete()
```

Comment:
This should be `delete_product` if we're back to use it.

Reply:
fixed